src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -238,7 +238,94 @@ public Task Get()
                 }
             }
         }
-        
+
+        [RavenAction("/admin/config-expiration-bundle", "POST", "/admin/config-expiration-bundle?name={databaseName:string}")]
+        public async Task ConfigExpirationBundle()
+        {
+            var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
+            string errorMessage;
+            if (ResourceNameValidator.IsValidResourceName(name, ServerStore.Configuration.Core.DataDirectory.FullPath, out errorMessage) == false)
+                throw new BadRequestException(errorMessage);
+
+            ServerStore.EnsureNotPassive();
+            TransactionOperationContext context;
+            using (ServerStore.ContextPool.AllocateOperationContext(out context))
+            {
+                var configurationJson = await context.ReadForMemoryAsync(RequestBodyStream(), "read-expiration-config");
+                var index = await ServerStore.ModifyDatabaseExpirationBundle(context, name, configurationJson);
+                await ServerStore.Cluster.WaitForIndexNotification(index);
+                ServerStore.NotificationCenter.Add(DatabaseChanged.Create(name, DatabaseChangeType.Update));
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.OK;
+
+                using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                {
+                    context.Write(writer, new DynamicJsonValue
+                    {
+                        ["ETag"] = index
+                    });
+                    writer.Flush();
+                }
+            }
+        }
+
+        [RavenAction("/admin/config-c-bundle", "POST", "/admin/config-versioning-bundle?name={databaseName:string}")]
+        public async Task ConfigVersioningBundle()
+        {
+            var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
+            string errorMessage;
+            if (ResourceNameValidator.IsValidResourceName(name, ServerStore.Configuration.Core.DataDirectory.FullPath, out errorMessage) == false)
+                throw new BadRequestException(errorMessage);
+
+            ServerStore.EnsureNotPassive();
+            TransactionOperationContext context;
+            using (ServerStore.ContextPool.AllocateOperationContext(out context))
+            {
+                var configurationJson = await context.ReadForMemoryAsync(RequestBodyStream(), "read-versioning-config");
+                var index = await ServerStore.ModifyDatabaseVersioningBundle(context, name, configurationJson);
+                await ServerStore.Cluster.WaitForIndexNotification(index);
```

Comment:
Need to wait on db, not server.

However, this node might not have that database setup on that node, if so, wait on the server only.